kyuubi-server/src/test/scala/org/apache/kyuubi/plugin/PluginLoaderSuite.scala

```diff
@@ -67,6 +67,19 @@ class PluginLoaderSuite extends KyuubiFunSuite {
     assert(clusterAConfFromCache.get("kyuubi.ha.namespace") == "kyuubi-ns-a")
     assert(clusterAConfFromCache.get("kyuubi.zk.ha.namespace") == null)
     assert(clusterAConfFromCache.size() == 5)
+
+    conf.set(KyuubiConf.SESSION_CONF_PROFILE, Seq("cluster-a", "cluster-b"))
+    val clusterABConf =
```

Comment:
This case does not seem to reflect the scenario of multiple profiles.

Reply:
In this case, since the value of `kyuubi.ha.namespace` is different, the value from `cluster-b` will replace the value from `cluster-a`. I will add more key-value pairs to `cluster-b` to refine this case.